src/test/java/com/opencart/tests/editAccount/ChangeEmail.java

```diff
@@ -0,0 +1,41 @@
+package com.opencart.tests.editAccount;
+
+import org.testng.annotations.Test;
+
+public class ChangeEmail extends LoginUsingValidData_Logout {
```

Comment:
I would recommend you to union ChangeEmail, ChangePassword, ChangeName tests into EditAccountTest class, also relocate Before and After method from LoginUsingValidData_Logout to new EditAccountTest class.